pom.xml

```diff
@@ -75,6 +75,11 @@
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
 			<version>2.2.0</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
```

Comment:
If I am not mistaken, spring already has an embedded Jackson parser. Does gson has better performance or other benefits in comparison?

Reply:
Yes, spring already has. But, the implementation of Json parser of spring convert strings to List\<Object\> or Map\<String, Object\> according to the [JsonParser documentation](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/json/JsonParser.html).

I wanted convert to JsonObject and compare the Json's, that's why I use Gson from Google.